app/src/main/java/edu/uc/forbesne/shoppingsidekick/ui/main/ProductListAdapter.kt

```diff
@@ -72,7 +72,8 @@ class ProductListAdapter( private val productList: ArrayList<Product>, mainViewM
         val btnIncrease: ImageButton = view_popup.findViewById(R.id.btnIncrease)
         val lblQuantity: TextView = view_popup.findViewById(R.id.etnQuantity)
 
-        Picasso.get().load(imgURL).into(prodImg);
+        val Picasso = null
+        val into: Any = Picasso.get().load(imgURL).into(prodImg);
```

Comment:
I was looking within the suggestions of Android Studio and through: https://kotlinlang.org/docs/coding-conventions.html#control-flow-statements on how to properly fix the Picasso and adding an explanation within the source.

Reply:
I really think this is going to return null, given how nulls operate in Kotlin.

Reply:
is into used anywhere?